Tests/web3swiftTests/localTests/TransactionsTests.swift

```diff
@@ -592,6 +592,18 @@ class TransactionsTests: XCTestCase {
         }
     }
 
+    func testDescription() async throws {
+        let vector = testVector[TestCase.eip1559.rawValue]
+        let jsonData = try XCTUnwrap(vector.JSON.data(using: .utf8))
+        let txn = try JSONDecoder().decode(CodableTransaction.self, from: jsonData)
+
+        let sut = String(describing: txn)
+
+        XCTAssertTrue(sut.contains("Transaction"))
+        XCTAssertTrue(sut.contains("from: "))
+        XCTAssertTrue(sut.contains("hash: "))
```

Comment:
Could you add some values after `from: ` and `hash: `?

Reply:
The from is being tested in the other test, I don't feel confortable doing it twice. Any idea?

Reply:
Hash assertion added